### ISSUE TITLE
'NoneType' object has no attribute 'unsqueeze' because mask is None in evaluating.

### ISSUE THREAD

**Issue by Scx123** [keonlee9420/Cross-Speaker-Emotion-Transfer#2]
When in evaluating,'self.training' is False,so mel_masks is None。Decoder encounters this error because mel_mask is None.

**Comment by freshwindy** [comment]
How did you solve this error?

**Comment by raikarsagar** [comment]
Any resolutions for this issue so far?

**Comment by keonlee9420** [comment]
Hi all, sorry for the late response. I just updated the repo with fixes of all issues (v0.2.0). Please check it out! Thanks.
